trident/ray_generator.py

```diff
@@ -161,14 +159,6 @@ def make_simple_ray(dataset_file, start_position, end_position,
         and :lines:='all', it will add every ion of every element up to Zinc.
         Default: None
 
-    :ionization_table: string, optional
```

Comment:
I removed this since I realized it is not actually used at all.  I guess we could deprecate and leave in, but I seriously doubt anyone ever manually sets the ionization table when generating a ray and just uses the default setup in their config file.

Reply:
I could see how this could be useful for making comparisons between multiple ionization tables. It's hard to know for sure that no one is using it. I'd say at least it should be deprecated first.